test/src/xquery/fn.xql

```diff
@@ -21,4 +41,17 @@ declare
 function fnt:normalize-unicode($normalization-form as xs:string) {
     count(string-to-codepoints(normalize-unicode($fnt:composed, $normalization-form))),
     count(string-to-codepoints(normalize-unicode($fnt:decomposed, $normalization-form)))
-};
\ No newline at end of file
+};
+
+declare
```

Comment:
BTW, [here](https://github.com/eXist-db/exist/blob/f7643dcb5d3c68c8c695b73d1ede6b0f8a8820b8/test/src/org/exist/xquery/XQueryTest.java#L1219) tests that have an external website dependency